mcs_benchmark_data/pipelines/commonsense_qa/kagnet_commonsense_qa_submission_file_names.py

```diff
@@ -2,12 +2,12 @@
 from dataclasses import dataclass
 from dataclasses_json import LetterCase, dataclass_json
 
-from mcs_benchmark_data._benchmark_submission_file_names import (
-    _BenchmarkSubmissionFileNames,
+from mcs_benchmark_data._benchmark_file_names import (
+    _BenchmarkFileNames,
 )
 
 
 @dataclass_json(letter_case=LetterCase.CAMEL)
 @dataclass(frozen=True)
-class KagnetCommonsenseQaSubmissionFileNames(_BenchmarkSubmissionFileNames):
+class KagnetCommonsenseQaSubmissionFileNames(_BenchmarkFileNames):
     submission_file_name: str = "dev_rand_split_kagnet_submission.jsonl"

```

Comment:
Some naming inconsistency here. In benchmarks you don't have a _file_name suffix.